crates/languages/src/css/highlights.scm

```diff
@@ -25,20 +19,29 @@
   "only"
 ] @operator
 
+[
+  (tag_name)
+  (nesting_selector)
+  (universal_selector)
+] @tag
+
+(universal_selector ("*") @tag)
```

Comment:
Isn't the universal selector always "*"? What is the difference between this and line 25?

Reply:
Ah I see you added it maybe because it was being overriden by `@operator` as per your comment? But now that line 25 is below the set of operators (3-20) this shouldn't be needed as this has higher specificity.

Reply:
It is also the multiplication symbol.

Reply:
Sure but like I said this shouldn't be needed given the presence of line 25 and line 25 being below the liens where the multiplication symbol is set.

Reply:
Tested on my zed. Removing this line will cause `*` to be highlighted as @operator.

Reply:
Gotcha. Well then maybe delete line 25 now instead since it is redundant / not helping.